src/components/Button.js

```diff
@@ -22,8 +33,11 @@ const Button = React.createClass({
     const styles = {
       component: {
         borderRadius: 2,
+        borderStyle: 'solid',
+        borderWidth: 1,
+        borderColor: 'transparent',
         display: 'inline-block',
-        padding: '7px 14px',
+        padding: '10px 14px',
```

Comment:
so this doesn't line up with the style guide now.  https://app.frontify.com/d/KHQZnlj1XMIO/ui-library-web#/ui-components/buttons  Is that ok?